libraries/helpers.rb

```diff
@@ -354,7 +354,7 @@ def url_for(pkg, base_url = 'ftp://ftp.software.ibm.com/aix/freeSoftware/aixtool
           'xft' => { version: '2.1.6-5', rpm: 'xft/xft-2.1.6-5.aix5.1.ppc.rpm' },
           'xpdf' => { version: '1.00-1', rpm: 'xpdf/xpdf-1.00-1.aix4.3.ppc.rpm' },
           'xpm-devel' => { version: '3.4k-8', rpm: 'xpm/xpm-devel-3.4k-8.aix6.1.ppc.rpm' },
-          'xpm' => { version: '3.4k-8', rpm: 'xpm/xpm-3.4k-8.aix5.2.ppc.rpm' },
+          # 'xpm' => { version: '3.4k-8', rpm: 'xpm/xpm-3.4k-8.aix5.2.ppc.rpm' },
```

Comment:
Same behaviour xpm key is duplicated
One must be removed  (not commented)

Installing Cookbook Gems:
Compiling Cookbooks...
/var/cache/chef/cookbooks/aix/libraries/helpers.rb:357: warning: key "xpm" is duplicated and overwritten on line 358